databricks/koalas/frame.py

```diff
@@ -7247,6 +7247,40 @@ def append(
 
         return cast(DataFrame, concat([self, other], ignore_index=ignore_index))
 
+    def combine_first(self, other: "DataFrame") -> "DataFrame":
+        if isinstance(other, ks.Series):
+            other = other.to_frame()
+
+        update_columns = list(
+            set(self._internal.column_labels).intersection(set(other._internal.column_labels))
+        )
+
+        update_sdf = self.join(
```

Comment:
Shall we consider using https://github.com/databricks/koalas/blob/master/databricks/koalas/utils.py#L80?

Reply:
Sure, makes sense. Will update the code and let you know.